content/discrete-math/concepts/proofs/terms/proof-by-cases/proof-by-cases.md

```diff
@@ -0,0 +1,70 @@
+---
+Title: 'Proof by Cases'
+Description: 'Proving a statement by showing that it holds in each of a set of mutually exhaustive cases.'
+Subjects:
+  - 'Computer Science'
+  - 'Discrete Math'
+Tags:
+  - 'Arithmetic'
+  - 'Logic'
+  - 'Logical'
+  - 'Math'
+CatalogContent:
+  - 'computer-science'
+  - 'paths/computer-science'
+---
+
+**Proof by Cases** is a technique used in mathematical proofs to demonstrate a statement by showing that it is true in each of a set of cases. These cases must be mutually exhaustive, meaning that they cover all possibilities, so that at least one of them must be true. By showing that the statement holds no matter which case is true, it is guaranteed that the statement is true.
+
+Proof by Cases is also sometimes known as _exhaustive proof_, which refers to the fact that the statement is proven in a set of mutually exhaustive cases.
+
+## Steps for Proof by Cases
+
+1. _Mutual exhaustion_: Show that there is a set of cases that is mutually exhaustive.
+2. _Prove each case_: Prove that the statement is true in each of the provided cases.
+
+## Example
+
+The statement below will be demonstrated by a Proof by Cases.
+
+```pseudo
+For any integer k, the product 3k^2 + k is even.
+```
+
+### Mutual exhaustion
```

Comment:
In the following example, can we rewrite so that we're not using the second-person pronoun, "we"?

Reply:
I can do this, but perhaps the Docs convention should have an exception for math proofs? Use of 'we' is a standard convention in proofs. Let me know what you decide.

Reply:
@noahpgordon I'm inclined to make this exception, but I'll defer to the other maintainers: @yangc95, @KTom101, @caupolicandiaz, what do you think?

Reply:
@SSwiniarski Shall we move forward with this?